dev/build.featureStart.base/src/com/ibm/ws/test/featurestart/features/FeatureFilter.java

```diff
@@ -67,6 +67,12 @@ public static String skipFeature(String shortName) {
             // location variable.
             return "Required variable 'wmqJmsClient.rar.location' is not set";
 
+        } else if (shortName.equalsIgnoreCase("mpHealth") ||
+                   shortName.equalsIgnoreCase("mpMetrics")){
```

Comment:
```suggestion
        } else if (!shortName.contains("-")){
```

This would be more robust and will enable the addition of new versionless features without the need to keep updating this code.